public/i18n.js

```diff
@@ -102,19 +102,21 @@ const locales = {
-    "ftm-index-where-to-find-ppe": "Look in your basement, your emergency kit, your bug out bag, your workbench for N95 masks.If you find surgical masks, those are also in short supply, and in many places are being used in place of N95s. All of these masks can save lives now if you get them into the hands of health care workers.",
-    "ftm-index-types-of-ppe": "Other types of PPE like disposable <span class=embold>booties</span>, <span class=embold>safety goggles</span>, and <span class=embold>disposable suits</span> are also in short supply. If you have these things or know someone who might (HVAC installers? Construction workers?) please share this site with them. Large donors can register at <a href=\"https://getusppe.org/\">getusppe.org/give</a>.",
-    "ftm-what-to-do-header": "Here's what you can do:",
-    "ftm-what-to-do-1": "Look at our list of sites below. You can filter by state or supplies needed.",
-    "ftm-what-to-do-2": "If there's a site near you, drop off following their instructions.",
-    "ftm-what-to-do-3": "If you are donating open packages, please put them in a ziploc bag with a note saying whether they have been used, whether anyone in your household has been sick or travelled outside the country, and the expiration date if known.",
-    "ftm-what-to-do-4": "If there's not a site near you, visit <a href=\"https://getusppe.org/give\">getusppe.org</a> to let them know what you have available.",
-    "ftm-add-donation-site": "To add a donation site, update information, or request we take down information, please <a class=\"add-donation-site-form\">fill in this form</a>. Your submission will be approved and updated as soon as possible.",
+    "ftm-index-where-to-find-ppe": "Maybe you bought N95 masks a few weeks ago or already have some in your house. Right now healthcare workers are running dangerously low on PPE like masks, hospital gowns, booties, and face shields to protect them while they risk their lives to save others.",
+    "ftm-index-types-of-ppe": "If you know construction workers, vets, scientists, or dentists, please share this link with them.",
+    "ftm-what-to-do-header": "How to Donate PPE",
+    "ftm-what-to-do-1": "<b>Use our map</b> below to find a hospital or health center near you accepting donations.",
+    "ftm-what-to-do-2": "<b>Follow their instructions</b> for what they accept and how to drop it off.",
+    "ftm-what-to-do-3": "<b>If you are donating opened packages</b>, please put them in a ziploc bag with a note saying whether they have been used, whether anyone in your household has been sick or travelled outside the country, and the expiration date if known.",
+    "ftm-add-donation-site-question": "Want to add a donation site? update or remove information?",
+    "ftm-add-donation-site-answer": "Please <a href='https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform'>fill out this form</a>. We will review it as soon as possible.",
```

Comment:
the donation form in the string needs to be `<a class=\"add-donation-site-form\">fill in this form</a>` to redirect correctly to the correct country form using this js: https://github.com/findthemasks/findthemasks/blob/39998ef9cbf3daacd07a5a9d3b42d00abcf7a1b1/public/locations-list-map.js#L307

Reply:
thanks

Reply:
so it should be 
```
"ftm-add-donation-site-answer": "Please <a class=\"add-donation-site-form\">fill in this form</a>. We will review it as soon as possible.",
``` ???

Reply:
👍 yup, exactly

Reply:
change committed